node_package/tests/ReactOnRails.test.js

```diff
@@ -19,10 +19,15 @@ describe('ReactOnRails', () => {
     });
     ReactOnRails.register({ R1 });
 
-    document.body.innerHTML = '<div id="root"></div>';
-    // eslint-disable-next-line no-underscore-dangle
-    const actual = ReactOnRails.render('R1', {}, 'root')._reactInternals.type;
```

Comment:
When we use a newer react version that supports the `Root API`, it returns the `ReactDomRoot` object, which doesn't have the `_reactInternals` property.